nanome/_internal/_process/_dssp.py

```diff
@@ -101,7 +101,7 @@ def __on_error(self, msg):
 
     def __dssp_done(self, result_code):
         if result_code != 0:
-            Logs.error("DSSP failed, code:", result_code)
+            Logs.warning("DSSP failed, code:", result_code)
```

Comment:
Is the user impacted by these errors? Like are they sitting waiting for something to happen? I'm skeptical of sweeping errors under the rug